source/common/protobuf/utility.h

```diff
@@ -367,7 +367,7 @@ class MessageUtil {
    *
    * @throw EnvoyException if the message does not unpack.
    */
-  static void unpackTo(const ProtobufWkt::Any& any_message, Protobuf::Message& message);
+  static void unpackToOrThrow(const ProtobufWkt::Any& any_message, Protobuf::Message& message);
```

Comment:
Would it make sense to get rid of this call altogether and use the THROW_IF_NOT_OK macto everywhere the throw is needed? It may be easier to search for throw callsites in the future.

Reply:
I'm 100% going to, but I'm trying to do it in a way that doesn't add THROW to more files because if I add one call site, others can add more.  If I fix excluded files one by one there will be fewer "leaks" I have to clean up.